.github/workflows/deploy-backend.yml

```diff
@@ -62,14 +62,16 @@ jobs:
           echo "$PRIVATE_KEY" > key.pem
           chmod 600 key.pem
 
-          # Copy compose files to EC2
+          # Copy compose/vault files to EC2
           scp -i key.pem -o StrictHostKeyChecking=no portal-backend-python/compose.yml $EC2_USER@$EC2_IP:~/
           scp -i key.pem -o StrictHostKeyChecking=no portal-backend-python/Caddyfile $EC2_USER@$EC2_IP:~/
+          scp -i key.pem -o StrictHostKeyChecking=no portal-backend-python/populate_env_prod.sh $EC2_USER@$EC2_IP:~/
 
           # Deploy on EC2
           ssh -i key.pem -o StrictHostKeyChecking=no $EC2_USER@$EC2_IP << EOF
             aws ecr get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin $ECR_REGISTRY
```

Comment:
unrelated to this pr but maybe we should move all the init logic into an init script